pkg/api/v1/status/atlasproject.go

```diff
@@ -34,6 +34,14 @@ func AtlasProjectSetPrivateEnpointsOption(privateEndpoints []ProjectPrivateEndpo
 	}
 }
 
+func AtlasProjectSetNetworkPeerOption(networkPeers *[]AtlasNetworkPeer) AtlasProjectStatusOption {
+	return func(s *AtlasProjectStatus) {
+		if networkPeers != nil {
```

Comment:
Here it's probably better to remove this line for cases where we actually want to reset the status. Like when someone decided to remove Network Peering we don't want status to stay frozen in the last state we had it enabled.

Reply:
Done